Ported TNTDoNotUpdate

DeadlyMC opened the pull request:
Added Patch Files for BlockTNT.java and enabled carpet rule in CarpetSettings.java

gnembon commented:
Sorry for late response - I was away. While I don't feel like this command is needed (in carpet save-X commands can be run from commandblock so every scenario should be reproduceable using vanilla save-Xs), I am not against it either. one question I have regarding the modified 1.12.json url. They both resolve, yet their content is different pointing into a different resource locations. Is the previous one wrong and if so why?

0x53ee71ebe11e commented:
I should have paid more attention to that. I didn't change anything in 1.12.json myself. Looks to me like that file get overwritten each time you run ./gradlew setupCarpetmod. Why is it even in the repository?

0x53ee71ebe11e commented:
What do you mean with your statement about the save-X commands? The /autosave command as implemented here has nothing in common with those and I don't see how I would use any of the vanilla commands as an autosave-detector, which is the main purpose of this command.

gnembon commented:
please revert that 1.12.json changes, so I can merge it.

prithamkjena committed:
Ported TNTDoNotUpdate

Added Patch Files for BlockTNT.java and enabled carpet rule in CarpetSettings.java
